packet/conn.go

```diff
@@ -53,19 +53,23 @@ type Conn struct {
 }
 
 func NewConn(conn net.Conn) *Conn {
+	return NewBufferedConn(conn, 65536) // 64kb
```

Comment:
keeping this `NewConn` function for compatibility reasons. As of now just the server code and most unit tests use this. The client close will be using `NewBufferedConn`.